### PR TITLE
fix: contains assertion bug; feat: add type_match, contained_by, string_equals assertion methods

### PR DESCRIPTION
**Pull request by bbx-winner**
fix: contains assertion bug, contains assertion should be customized.
feat: add plentiful response assertion methods: type_match, contained_by, string_equals assertion methods.

### CODE REVIEW

File: internal/builtin/assertion.go [hunk 1]
```diff
@@ -83,3 +86,20 @@ func convertInt(value interface{}) (int, error) {
 		return 0, fmt.Errorf("unsupported int convertion for %v(%T)", v, v)
 	}
 }
+
+// Contains assert whether actual
```
Comment (debugtalk) [inline internal/builtin/assertion.go:90]: @bbx-winner 补充完整

File: internal/builtin/assertion.go [hunk 2]
```diff
@@ -83,3 +86,20 @@ func convertInt(value interface{}) (int, error) {
...
+func Contains(t assert.TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool {
+	return assert.Contains(t, actual, expected, msgAndArgs)
```
Comment (debugtalk) [inline internal/builtin/assertion.go:92]: 这个看上去跟 contained_by 是一样的？

File: internal/builtin/assertion.go [hunk 3]
```diff
@@ -83,3 +86,20 @@ func convertInt(value interface{}) (int, error) {
...
+func EqualString(t assert.TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool {
+	if !assert.IsType(t, "string", actual, msgAndArgs) {
+		return false
+	}
+	if !assert.IsType(t, "string", expected, msgAndArgs) {
+		return false
+	}
+	actualString := actual.(string)
+	expectedString := expected.(string)
+	return assert.True(t, strings.EqualFold(actualString, expectedString), msgAndArgs)
```
Comment (debugtalk) [inline internal/builtin/assertion.go:104]: 这里 EqualFold 忽略大小写了？建议还是分开吧

File: internal/builtin/assertion.go (hunk 2 above)
Comment (bbx-winner) [reply internal/builtin/assertion.go:92]: 区别就是 actual 和 expected 的顺序与 contained_by 不同

File: internal/builtin/assertion.go (hunk 1 above)
Comment (bbx-winner) [reply internal/builtin/assertion.go:90]: done

File: internal/builtin/assertion.go (hunk 3 above)
Comment (bbx-winner) [reply internal/builtin/assertion.go:104]: 如果不忽略大小写严格匹配，就和之前的 assert.EqualValues 功能一样了；string_equals 断言函数是专门针对字符串的

**Pull request merged**